src/Http/Controllers/Auth/LoginController.php

```diff
@@ -23,13 +23,13 @@ public function store(): View
     public function login(LoginUserRequest $request, UserLoginService $service): RedirectResponse|View
     {
         try {
-            $service->loginUser($request->get("email"), $request->get("password"));
+            $service->checkUser($request->get("email"), $request->get("password"));
         } catch (AuthenticationException $exception) {
             return view("user.login")
                 ->with("error", $exception->getMessage());
         }
 
-        return redirect()->route("home");
+        return redirect()->route("2fa.index")->with(["email" => $request->get("email")]);
```

Comment:
Users without 2FA are also redirected to the 2FA site?

Reply:
now they don't